conntrack_test.go

```diff
@@ -479,6 +479,65 @@ func TestConntrackFilter(t *testing.T) {
 		t.Fatalf("Error, there should be an exact match, v4:%d, v6:%d", v4Match, v6Match)
 	}
 
+	// SrcIPNet filter
+	filterV4 = &ConntrackFilter{}
+	filterV4.AddIPNet(ConntrackOrigSrcIP, &net.IPNet{IP: net.ParseIP("10.0.0.1"), Mask: net.CIDRMask(12, 32)})
```

Comment:
and there's a function for that too

```suggestion
	filterV4.AddIPNet(ConntrackOrigSrcIP, ParseIPNet("10.0.0.1/12"))
```

though presumably you mean `"10.0.0.0/12"` anyway?

actually all of the ipnet filters here look wrong; they should have 0s in the low bits